test/services/onlinetesting/Phase3TestServiceSpec.scala

```diff
@@ -131,7 +186,10 @@ class Phase3TestServiceSpec extends PlaySpec with MockitoSugar with ScalaFutures
 
     val testInterviewId = 123
     val testTimeNow = DateTime.parse("2016-10-01T00:00:01Z")
-    val testExpiryTime = testTimeNow.plusDays(7)
+    val unexpiredTestExpiryTime = DateTime.parse("2016-11-01T00:00:01Z")
```

Comment:
Can't we mock dateTimeFactory and return always date which we know (localdate.now()). It would make the tests not bound to this exact date.

If it will be a lot of work leave it as it is.